plugins/pulp_rpm/plugins/importers/yum/repomd/primary.py

```diff
@@ -272,10 +262,7 @@ def _process_file_element(file_element):
     :return: file information dictionary
     :rtype: dict
     """
-    # NOTE the use of deepcopy relies on cpython's very sensible policy of never
-    # duplicating string literals, this may not hold up in other implementations
-    # the python interpreter.
-    file_info = deepcopy(FILE_INFO_SKEL)
```

Comment:
Good point! The one which is used and has the same names comes from metedata.py, so this skeleton can be removed.